### ISSUE TITLE
O'FAIRe tool, question R1.3Q2

### ISSUE THREAD

**Issue by martinaPulieri** [lifewatch-eric/ecoportal_web_ui#48]
We should discuss how to set up the question “R1.3Q2 : Is the ontology included in a specific community set or group?” to be specific for each portal.

![image](https://github.com/lifewatch-eric/ecoportal_web_ui/assets/149166663/4846ab81-2e6d-4043-a45e-2e958b4e2fe5)

**Comment by jonquet** [comment]
Yes indeed. 
My proposition would be to have on the portal side a property for groups: 
- FairLevel that will be 1,2 or 3 
- on the O'FAIRe side evaluate the points based on the FairLevel of a group
- adapt O'FAIRe explanation message
- include the FairLevel explanation in the UI planned to describe the Groups

**Comment by syphax-bouazzouni** [comment]
@jonquet in Agroportal you need to define what OBO Foundry is, compared to  OBO Libray, currently it only sets hard-coded ontologies. 

![image](https://github.com/user-attachments/assets/52755e28-0ca0-4d77-b5d4-e740afb18d98)

Maybe create a new group called OBO Library, besides the current one OBO Foundry.

**Comment by jonquet** [comment]
Not sure I am following the demande here.
Our group is 
"name": "OBO Foundry",
"description": "This group includes ontologies currently available within the obofoundry.org platform.",

We do not have a group "OBO Library"

**Comment by syphax-bouazzouni** [comment]
What is the diffrence between this two ? in term of groups. 
![image](https://github.com/user-attachments/assets/966fa4c4-4d0e-427a-bb3f-f20fa559c0f2)

**Comment by jonquet** [comment]
ahh... this bring us back to a distinction that the OBOFoundry people were originally doing. There were a few ontologies that had the status "foundry" while the other had the status "library" and the former where the original, more solid and qualitative ones. 

In the implmeetation of this feature (R1.3Q2) my propositon is to now rely on an informaiton provided by the portal, assessing the FAIR level of the groups (whatever they are) the ontology is in. 
In other words we would have a property 
groupFairLevel=[3, 2, 2]
telling us that the ontology is in 3 groups one of fairlevel 3 and 2 of fairlevel 2. O'FAIRe on its side will only have to do the maths to determines the mount points given (betweem 0 and 20)

**Comment by jonquet** [comment]
Waiting for a a longer term, this can be fixed only on the EcoPortal side... considering contributing collaboratively to O'FAIRe code is not that trivial.

**Comment by syphax-bouazzouni** [comment]
I did a temporary fix for Ecoportal as they don't use groups, now we show the message, no more referencing to the AgroPortal groups 
![image](https://github.com/user-attachments/assets/0a9f34a2-3cc8-4c0d-8af5-8aac244bf4b1)
I opened these two issues https://github.com/ontoportal-lirmm/bioportal_web_ui/issues/892 and  https://github.com/agroportal/fairness/issues/35, for future iteration on a generic and definitive solution.
